provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving cronjob
+  copy:
```

Comment:
The reason I used the copy module instead of the cron module is that I couldn't verify that it had actually created the job. I couldn't work out where it was making the addition. I suspect a closer reading of the module docs will reveal this :)